app/src/main/java/com/kinandcarta/create/proxytoggle/feature/manager/view/ProxyManagerFragment.kt

```diff
@@ -15,16 +17,14 @@ import dagger.hilt.android.AndroidEntryPoint
 class ProxyManagerFragment : Fragment() {
 
     companion object {
-        fun newInstance() =
-            ProxyManagerFragment()
-
-        const val FI_ADDRESS = "192.168.1.215"
-        const val FI_PORT = "8888"
+        fun newInstance() = ProxyManagerFragment()
```

Comment:
I don't really understand the point of this? I know AS adds it by default if you use a template but I don't see the benefit

Reply:
Before the navigation component and safeArgs, you had to manually send the arguments to the Fragment from the origin and then collect them once in the Fragment.
Consider this situation in pseudo-code:
```kotlin
fun newInstance(id: Int, name: String) = YourFragment().withArguments(ARG_ID = id, ARG_NAME = name)
```
This pattern allows you to use a static method (therefore, you don't need an instance of the fragment) with all the arguments you'd like to send, not caring about the actual argument:
```kotlin
val fragment = YourFragment.newInstance(1, "Kane")
```